mux/spec/cli.md

```
@@ -0,0 +1,167 @@
+| Verb | Status | Required flags/args | Optional flags | Human stdout |
+| --- | --- | --- | --- | --- |
+| `identify` | implemented | none | global flags | one metadata line |
+| `list-workspaces` | implemented | none | global flags | tree lines |
+| `send` | implemented | `--surface <id>` | `--text <text>`, `--bytes <base64>` | none |
+| `read-screen` | implemented | `--surface <id>` | none | screen text |
+| `vt-state` | implemented | `--surface <id>` | none | `cols=<n> rows=<n> data=<base64>` |
+| `new-tab` | implemented | none | `--pane <id>`, `--cwd <path>`, `--cols <n> --rows <n>` | surface id |
+| `new-browser-tab` | implemented | `--url <url>` | `--pane <id>`, `--cols <n> --rows <n>` | surface id |
+| `new-workspace` | implemented | none | `--name <name>`, `--cols <n> --rows <n>` | surface id |
+| `new-screen` | implemented | none | `--workspace <id>`, `--cols <n> --rows <n>` | surface id |
+| `split` | implemented | `--pane <id> --dir right|down` | `--cols <n> --rows <n>` | surface id |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)